lambda/metric/index.js

```diff
@@ -0,0 +1,67 @@
+const AWS = require('aws-sdk');
+const { runScan } = require('../dynamoUtil');
+const { sendResponse } = require('../responseUtil');
+const { checkPermissions } = require('../permissionUtil');
+
+exports.handler = async (event, context) => {
+  console.log('Metric', event);
+  console.log('event.queryStringParameters', event.queryStringParameters);
+
+  let queryObj = {
+    TableName: process.env.TABLE_NAME
```

Comment:
Please use the `TABLE_NAME` constant from `dynamoUtil.js` instead of `process.env`.   The constant has a default value.

Reply:
d3fda1e